### DISCUSSION TITLE
Add option to "Save and use new asset"

### DISCUSSION THREAD

**Discussion by thupsi** [craftcms/cms#9687] [Ideas] [answered]
When a user edits an asset, she is given the option to "save as a new asset". When this happens in the context of an entry edit page (i.e. by opening the slideout for that asset), I think it would make sense to also have the option to **select/use the new asset** in place of the old.

Example: I'm editing an entry with a related image and I would like to use a cropped version of that image. I have to:
- Open asset slideout
- Click edit
- Crop the existing image
- Choose "save as a new asset"
- Close the slideout
- Remove the original from the asset field 
- Find and select the new asset

The last two steps seem kind of unnecessary.

**Comment by brandonkelly** [comment, marked as answer]
This was previously requested by #8974. Follow along there for updates.
